main.html

```diff
@@ -0,0 +1,217 @@
+<!DOCTYPE html>
```

Comment:
Why did you add this new file? There is no such need I guess.

Reply:
I thought since I closed my first PR all my changes are gone. So I uploaded the main.html with my changes in the footer.

Reply:
No please delete this.